src/cmake/axom-config.cmake.in

```diff
@@ -188,6 +188,13 @@ if(NOT AXOM_FOUND)
   # mfem
   if(AXOM_USE_MFEM)
     set(AXOM_MFEM_DIR     "@MFEM_DIR@")
+    if(EXISTS "${AXOM_MFEM_DIR}/lib/cmake/mfem/MFEMConfig.cmake")
+      set(MFEM_DIR ${AXOM_MFEM_DIR})
```

Comment:
I don't think this line is needed since AXOM_MFEM_DIR was just set to MFEM_DIR, so they should already be the same.

Reply:
Peter reminded me that was at the time this file was configured and MFEM_DIR is no longer set when this file is used.